Makefile.rpmbuilder

```diff
@@ -92,16 +92,14 @@ endif
 
 # this '[ -n "$(1)" ] && ' is to suppress "call" at function definition
 spec_packages = $(shell cd $(ORIG_SRC) && [ -n "$(1)" ] && \
-	if [ 0`stat -c %Y $(OUTPUT_DIR)/$(notdir $(1)).list 2>/dev/null` -ge \
-			0`git log -1 --pretty=format:%ct` ]; then \
+	if [ 0$(stat -c %Y $(OUTPUT_DIR)/$(notdir $(1))-$(PACKAGE_SET).list 2>/dev/null) -ge 0$(git log -1 --pretty=format:%ct) ]; then \
```

Comment:
If you change the .list file name, you need to change also the place that creates it - in Makefile-legacy and Makefile-mock. Right now you simply disabled this mechanism, so it will always choose the fallback below (query-spec).